tests/foreman/destructive/test_clone.py

```diff
@@ -15,13 +15,18 @@
 
 from robottelo import constants
 from robottelo.config import settings
-from robottelo.hosts import Satellite
+from robottelo.hosts import Satellite, get_sat_rhel_version
 
 SSH_PASS = settings.server.ssh_password
 pytestmark = pytest.mark.destructive
 
 
 @pytest.mark.e2e
+@pytest.mark.parametrize(
+    "sat_ready_rhel",
+    [8, 9] if get_sat_rhel_version().major < 9 else [9],
+    indirect=True,
+)
```

Comment:
@lpramuk I'm not 100% sure but you'll probably  have to fix [sat_ready_rhel.download_repofile](https://github.com/SatelliteQE/robottelo/pull/14856/files#diff-0136ad9f3a21adf5716d5013efd837789362d2624fb5c83f55a226650e0c0470R91) part as well. I think it'll use rhel version of `target_satellite` or from server config and might get the wrong rhel version repo. To avoid this it's better to pass the required RHEL version explicitly.

Reply:
`download_repofile` doesn't support passing rhel version atm so you'll have to find another way or update it to support that.😅

Reply:
This bubbles thru and gets to `def _os_release(self):` which reads `/etc/release` 
Safe I'd say :-)

Reply:
ah cool